chainer/device_resident.py

```diff
@@ -14,6 +15,29 @@
 import chainerx
 
 
+def _warn_to_gpu(dst, arr, legacy):
```

Comment:
How about moving this into `_ToDeviceVisitor` as a method?
It's only used in the class.

Reply:
How about this comment?